src/reducer.ts

```diff
@@ -349,6 +360,10 @@ export default (oldState: State, action: Action): State => {
     }
   }
 
+  // The userMap is always saved so if somebody writes new functionality that edits the userMap, they don't have to remember
+  // to save it.
+  localStorage.setItem('userMap', JSON.stringify(state.userMap))
```

Comment:
I'm mildly in favor of still limiting this to the UserMapAction codepath?

I can anticipate other functionality that edits the userMap (see https://github.com/lazerwalker/azure-mud/issues/112), but I think it's unlikely there would be changes to the userMap functionality where it doesn't still make sense to dispatch a UserMapAction. And given that, it does seem wasteful to keep writing this every reduce.